static-site/src/components/Footer/sitemap.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import styled from "styled-components";
```

Comment:
FWIW, I'd like us to move away from styled components over time. This isn't a request to stop using it in this PR - we use it everywhere at the moment, so staying consistent is great - but just a heads up as to my thinking of where we'll take this over time.

Reply:
Could you write up an issue for this? I remember you also brought it up somewhere else with more reasoning but I can't seem to find it.

Reply:
It's really part of that larger discussion issue that I can't find at present -- what frontend do we want? That discussion involves styles, how data is passed between client & server etc.

Reply:
Issue created: #963